todo_app/templates/index.html

```diff
@@ -6,63 +6,78 @@
-              </select>
-            </li>
-          </form>
+          {% if 'writer' == view_model.user_role %}
+            <form action = "/items/{{item.card_id}}" method = "get">
+              <li>{{item.name}} 
+                <select name="status" id="status" onchange="this.form.submit()">
+                  <option value="To Do"{% if 'To Do' == item.status %} SELECTED{% endif %}>To Do</option>
+                  <option value="Doing"{% if 'Doing' == item.status %} SELECTED{% endif %}>Doing</option>
+                  <option value="Done"{% if 'Done' == item.status %} SELECTED{% endif %}>Done</option>
+                </select>
+              </li>
+            </form>
+          {% else %}
+            <li>{{item.name}}</li> 
+          {% endif %}
         {% endfor %}
       </ul>
     </div>
     <div class="col">
       <h2>Doing</h2>
       <ul class="list-group mb-4">
         {% for item in view_model.doing_items %}
-          <form action = "/items/{{item.card_id}}" method = "get">
-            <li>{{item.name}} 
-              <select name="status" id="status" onchange="this.form.submit()">
-                <option value="To Do"{% if 'To Do' == item.status %} SELECTED{% endif %}>To Do</option>
-                <option value="Doing"{% if 'Doing' == item.status %} SELECTED{% endif %}>Doing</option>
-                <option value="Done"{% if 'Done' == item.status %} SELECTED{% endif %}>Done</option>
-              </select>
-            </li>
-          </form>
+          {% if 'writer' == view_model.user_role %}
+            <form action = "/items/{{item.card_id}}" method = "get">
+              <li>{{item.name}} 
+                <select name="status" id="status" onchange="this.form.submit()">
+                  <option value="To Do"{% if 'To Do' == item.status %} SELECTED{% endif %}>To Do</option>
+                  <option value="Doing"{% if 'Doing' == item.status %} SELECTED{% endif %}>Doing</option>
+                  <option value="Done"{% if 'Done' == item.status %} SELECTED{% endif %}>Done</option>
+                </select>
+              </li>
+            </form>
+          {% else %}
+            <li>{{item.name}}</li> 
+          {% endif %}
         {% endfor %}
       </ul>
     </div>
     <div class="col">
       <h2>Done</h2>
       <ul class="list-group mb-4">
         {% for item in view_model.done_items %}
-          <form action = "/items/{{item.card_id}}" method = "get">
-            <li>{{item.name}} 
-              <select name="status" id="status" onchange="this.form.submit()">
-                <option value="To Do"{% if 'To Do' == item.status %} SELECTED{% endif %}>To Do</option>
-                <option value="Doing"{% if 'Doing' == item.status %} SELECTED{% endif %}>Doing</option>
-                <option value="Done"{% if 'Done' == item.status %} SELECTED{% endif %}>Done</option>
-              </select>
-            </li>
-          </form>
+          {% if 'writer' == view_model.user_role %}
```

Comment:
These three change blocks are a reminder that there's a fair amount of duplicated code here - not particularly important but something we could improve, e.g. [by using a macro](https://stackoverflow.com/questions/21275375/reuse-a-block-of-code-in-several-places-in-jinja2/21275796)